### ISSUE TITLE
We need to test that we can handle multiple multiConnects with multiple TCP sessions per multiConnect at the native level

### ISSUE THREAD

**Issue by yaronyg** [thaliproject/Thali_CordovaPlugin#1074]
In theory we have node tests that are to be written that will open up multi simultaneous MCSessions with multiple devices and then open multiple TCP streams with each device, send data and confirm it was received (usually via an echo test). But we really need to test this at the native level first so we can shake out any issue in an easier to debug environment.

**Comment by baydet** [comment]
Started implementation of echo test. I use AppContext's interface to establish MCPF connection. 
I'm finishing. Now I'm finishing multipeer connectivity part. As soon as #881 will be merged into vNext it will be possible to use CocoaAsyncSocket for TCP connections management.

**Comment by baydet** [comment]
blocked on thaliproject/Thali_CordovaPlugin#881
